simulator/src/modules/Splitter.js

```diff
@@ -38,7 +38,7 @@ export default class Splitter extends CircuitElement {
         */
         this.rectangleObject = false;
 
-        this.bitWidthSplit = bitWidthSplit || prompt('Enter bitWidth Split').split(' ').filter((lambda) => lambda !== '').map((lambda) => parseInt(lambda, 10) || 1);
+        this.bitWidthSplit = bitWidthSplit || (prompt('Enter bitWidth Split') || '1').split(' ').filter((lambda) => lambda !== '').map((lambda) => parseInt(lambda, 10) || 1);
```

Comment:
Thank you for this, I have a few concerns with this
I am not sure if this is helpful. If the splitter has a bitwidth of `n`. Then we want all splits to add up to `n`. If this does not happen, then the splitter behaves in an unsoundly.

You can maybe change this to have `'1'.repeat(bitWidth)`?

Reply:
Thanks for the advice. I went and changed it to `'1 '.repeat((bitWidth || 1) - 1) + '1'`. Basically, this will give it the default behavior of splitting a bus of size `n` to `n` number of splits if user chooses to cancel the prompt or enters nothing.